src/index.d.ts

```diff
@@ -48,7 +48,7 @@ declare module 'react-native-screens' {
   export type ScreenReplaceTypes = 'push' | 'pop';
 
   export interface ScreenProps extends ViewProps {
-    active?: 0 | 1 | Animated.AnimatedInterpolation;
+    active?: 0 | 1 | 2 | Animated.AnimatedInterpolation;
```

Comment:
I'd perhaps rename this to something like "activityState" and keep old "active" prop for compatibility while just transforming its value.

Would also rename "shouldUseNewImplementation" to something that gives more clue (will "new" still be new in five months?). So given we rename prop to "activityState" then the other one can be named "shouldUseActivityState"